app/Services/AccountService.php

```diff
@@ -1,26 +1,25 @@
 <?php
 
 namespace App\Services;
-
-use App\Http\Controllers\AccountController;
 use App\Models\Wishlist;
+use App\Models\User;
+Use Illuminate\Support\Facades\DB;
+Use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Collection;
 use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
 
 class AccountService
 {
 
-    private function getWishlist():array
+    private function getWishlist(): Collection
     {
-        return
-        [
-            ['title'=>'Catan'],
-            ['title'=>'Risk'],
-            ['title'=>'Ticket to Ride']
-        ];
+        // WTF DO I PUT HERE????????????????
```

Comment:
Are you supposed to replace the directly defined titles with information from the Games table?